### ISSUE TITLE
Getting field graph statistical functions in Reports is broken

### ISSUE THREAD

**Issue by edmundoa** [Graylog2/graylog2-server#6977]
The mapping to get a human readable statistical function for field graphs widgets in Reports doesn't handle all used statistical functions. For stacked graphs, this may throw an error while generating a Report:

```
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at WidgetDescription.jsx:104
    at Array.map (<anonymous>)
    at WidgetDescription._multipleSeriesDescription (WidgetDescription.jsx:103)
    at WidgetDescription.render (WidgetDescription.jsx:136)
    at finishClassComponent (react-dom.development.js:14741)
    at updateClassComponent (react-dom.development.js:14696)
```

For regular field graphs we guard against an `undefined` value and we don't render the statistical function:
<img width="647" alt="Screenshot 2019-12-11 at 14 53 37" src="https://user-images.githubusercontent.com/716185/70627311-4a5a7a80-1c26-11ea-8932-3c27c9127c01.png">

For supported statistical functions, this is what we render:
<img width="616" alt="Screenshot 2019-12-11 at 14 54 00" src="https://user-images.githubusercontent.com/716185/70627540-b341f280-1c26-11ea-9b6a-8800ddb71063.png">

## Expected Behavior
All used statistical functions should be supported in Reports. Even if that's not the case, this should not stop us from rendering the Report.

## Current Behavior
Not all used statistical functions are supported in Reports, and they may stop the report from being generated.

## Steps to Reproduce (for bugs)
1. Create a stack graph using `total` as statistical function for at least one of the series
2. Add the graph to a dashboard
3. Add the widget to a report
4. Render the report

## Your Environment
* Graylog Version: Graylog-3.1.3

**Comment by bernd** [comment]
This has been fixed in https://github.com/Graylog2/graylog-plugin-enterprise/pull/1348.

**Comment by edmundoa** [comment]
Reopening, since we still need to verify if this needs to be fixed in 3.2. At least we should incorporate changes making the widget rendering more resilient.

**Comment by bernd** [comment]
@edmundoa Thanks! That means we can move it from the backports project into the search 3.2 project and also update the milestone to 3.2.0?
